recipe/recipe.yaml

```diff
@@ -0,0 +1,139 @@
+      - $CXX 05-eplus_eminus.cc -o 05-eplus_eminus $CXXFLAGS $LDFLAGS -lfastjet
+      - ./05-eplus_eminus < data/single-ee-event.dat
+      - $CXX 06-area.cc -o 06-area $CXXFLAGS $LDFLAGS -lfastjet
+      - ./06-area < data/single-event.dat
+      - $CXX 07-subtraction.cc -o 07-subtraction $CXXFLAGS $LDFLAGS -lfastjettools -lfastjet
+      - ./07-subtraction < data/Pythia-Zp2jets-lhc-pileup-1ev.dat
+      - $CXX 08-selector.cc -o 08-selector $CXXFLAGS $LDFLAGS -lfastjet
+      - ./08-selector < data/single-event.dat
+      - $CXX 09-user_info.cc -o 09-user_info $CXXFLAGS $LDFLAGS -lfastjet
+      - ./09-user_info < data/Pythia-dijet-ptmin100-lhc-pileup-1ev.dat
+      - $CXX 10-subjets.cc -o 10-subjets $CXXFLAGS $LDFLAGS -lfastjet
+      - ./10-subjets < data/single-event.dat
+      - $CXX 11-filter.cc -o 11-filter $CXXFLAGS $LDFLAGS -lfastjettools -lfastjet
+      - ./11-filter < data/single-event.dat
+      - $CXX 12-boosted_higgs.cc -o 12-boosted_higgs $CXXFLAGS $LDFLAGS -lfastjettools -lfastjet
+      - ./12-boosted_higgs < data/HZ-event-Hmass115.dat
+      - $CXX 13-boosted_top.cc -o 13-boosted_top $CXXFLAGS $LDFLAGS -lfastjettools -lfastjet
+      - ./13-boosted_top < data/boosted_top_event.dat
+      - $CXX 14-groomers.cc -o 14-groomers $CXXFLAGS $LDFLAGS -lfastjettools -lfastjet
+      - ./14-groomers < data/boosted_top_event.dat
+    files:
+      source:
+        - example/fastjet_example.cc
+        - example/fastjet_areas.cc
+        - example/01-basic.cc
+        - example/02-jetdef.cc
+        - example/03-plugin.cc
+        - example/04-constituents.cc
+        - example/05-eplus_eminus.cc
+        - example/06-area.cc
+        - example/07-subtraction.cc
+        - example/08-selector.cc
+        - example/09-user_info.cc
+        - example/10-subjets.cc
+        - example/11-filter.cc
+        - example/12-boosted_higgs.cc
+        - example/13-boosted_top.cc
+        - example/14-groomers.cc
+        - example/data/single-event.dat
+        - example/data/single-ee-event.dat
+        - example/data/Pythia-Zp2jets-lhc-pileup-1ev.dat
+        - example/data/Pythia-dijet-ptmin100-lhc-pileup-1ev.dat
+        - example/data/HZ-event-Hmass115.dat
+        - example/data/boosted_top_event.dat
+    requirements:
+      run:
+        - ${{ compiler('cxx') }}
+
+about:
+  homepage: https://gitlab.com/fastjet/fastjet
+  summary: 'FastJet: jet finding in pp and e+e- collisions'
+  description: |
+    FastJet is a software package for jet finding in pp and e+e- collisions.
+    It includes fast native implementations of many sequential recombination
+    clustering algorithms, plugins for access to a range of cone jet finders
+    and tools for advanced jet manipulation.
+  license: GPL-2.0-or-later
+  license_family: GPL
+  license_file: COPYING
+  repository: https://gitlab.com/fastjet/fastjet
+  documentation: https://fastjet.fr/
+
+extra:
+  recipe-maintainers:
+    - kratsg
+    - matthewfeickert
```

Comment:
@lgray, no pressure, but feel free to add yourself here as well if you'd like.

Reply:
I'll add myself here once we get the cmake release of fastjet. I'd like to do the change over for that.

Reply:
Ah, sure that makes sense. Let's way for fastjet to churn.